src/_gettsim/transfers/rente.py

```diff
@@ -91,6 +63,49 @@ def ges_rente_vor_grundr_m(
     return out
 
 
+@dates_active(end="2020-12-31")
+def ges_rente_m(
+    ges_rente_zugangsfaktor: float,
+    entgeltp_update: float,
+    rentenwert: float,
+    rentner: bool,
+) -> float:
+    return ges_rente_vor_grundr_m(
```

Comment:
Why don't we use just `ges_rente_vor_grundr_m` as argument?

Reply:
Great point! Done in 816efc9.